dll/win32/browseui/shellbrowser.cpp

```diff
@@ -120,6 +120,7 @@ static const unsigned int                   folderOptionsPageCountMax = 20;
 static const long                           BTP_DONT_UPDATE_HISTORY = 0;
 static const long                           BTP_UPDATE_CUR_HISTORY = 1;
 static const long                           BTP_UPDATE_NEXT_HISTORY = 2;
+static const long                           BTP_ACTIVATE_NOFOCUS = 0x04;
```

Comment:
```suggestion
static const long                           BTP_ACTIVATE_NOFOCUS = 4;
```
(to keep with the rest of the constants)
Are these BTP_***  flags of sorts?

Reply:
The existing ones seem to be treated as flags yes. The one I added is a flag for sure.